woocommerce/google-listings-and-ads
# Setup Ads Step 2: Display Free Ad Credit Value by Country Modal

### Changes proposed in this Pull Request:

Closes #375 .

This PR allows the users to view the free ad credit value by country in a modal popup in the Setup Ads Step 2.

### Screenshots:

![Free ad credit by country modal 2](https://user-images.githubusercontent.com/417342/112437683-2a4ebc00-8d82-11eb-9b70-217b93e18350.gif)

### Detailed test instructions:

Pre-requisite:
Make sure that you are using a Google Ads sub-account that is newly created within the past 30 days. It must not be disconnected and reconnect. This is the eligibility criteria for the free ad credit.

1. Open https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-ads
2. Proceed to Step 2.
3. You should see the free ad credit notice.
4. Click on the here link.
5. A modal should show up as per the screenshot above.

### Changelog Note:

- Display free ad credit by country modal.
